src/bundle/Templating/Twig/DateTime/DateTimeExtension.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace EzSystems\EzPlatformAdminUiBundle\Templating\Twig\DateTime;
+
+use Twig\Extension\AbstractExtension;
+use Twig\TwigFilter;
+
+class DateTimeExtension extends AbstractExtension
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function getFilters(): array
+    {
+        return [
+            new TwigFilter('ez_datetime', [DateTimeRuntimeExtension::class, 'format']),
```

Comment:
I can't exactly recall if you need a static methods for twig filters but I suppose it accepts any callable. Static method doesn't make too much sense for me here. Especially because `DateTimeRuntimeExtension` is not making anything.